dart/lib/src/protocol/sentry_user.dart

```diff
@@ -64,33 +74,62 @@ class SentryUser {
   ///
   /// These keys are stored as extra information but not specifically processed
   /// by Sentry.
+  final Map<String, dynamic>? data;
+
+  @Deprecated('Will be removed in v7. Use [data] instead')
   final Map<String, dynamic>? extras;
 
+  /// Approximate geographical location of the end user or device.
```

Comment:
```suggestion
  /// Approximate geographical location of the end user or device.
  /// 
  /// The geolocation is automatically inferred by Sentry.io if the [ipAddress] is set.
  /// Sentry however doesn't collect the [ipAddress] automatically because it is PII.
```